libp2p/protocols/kademlia/kademlia.nim

```diff
@@ -91,7 +154,7 @@ proc findNode*(
 
       pendingFutures[peer] = kad
         .sendFindNode(peer, addrTable.getOrDefault(peer, @[]), targetId)
-        .wait(5.seconds)
+        .wait(chronos.seconds(5))
```

Comment:
Required as both times and chronos has seconds imported. either import peicemeal, or keep as is

Reply:
Import peicemeal (idk what that means but guessing it means we get `5.seconds` in the end which is what I'd like to see rofl)

Reply:
https://github.com/vacp2p/nim-libp2p/issues/1605